packages/pie-models/src/pie/multiple-choice/index.ts

```diff
@@ -32,6 +32,11 @@ export interface MultipleChoicePie extends PieModel {
   /** Determines if prompt should show */
   promptEnabled?: boolean;
 
+  /** Indicates the layout of choices for player
+   * @default: true
+   */
+  verticalLayout?: boolean;
```

Comment:
I see you added **verticalMode** property, but in docs you've named it **verticalLayout**.

@dianagociu , can we please use verticalMode in the docs as well?